TILApp/Views/CommunityView/CommunityViewController.swift

```diff
@@ -106,15 +105,57 @@ extension CommunityViewController: UITableViewDataSource {
                     completion(state)
                 }
             }
-
             let webViewController = WebViewController(webView: WKWebViewWarmer.shared.dequeue()).then {
                 $0.navigationItem.rightBarButtonItem = UIBarButtonItem(customView: likeButton)
                 $0.hidesBottomBarWhenPushed = true
                 $0.url = post.url
             }
             navigationController?.pushViewController(webViewController, animated: true)
         }
+        cell.customCommunityTILView.makeMenuItems = post.user.id != AuthViewModel.shared.user?.id ? { [weak self] in
+            let reportAction = UIAction(title: "차단하기", image: UIImage(systemName: "eye.slash")) { [weak self] _ in
+                guard let self else {return}
+                UserViewModel.shared.blockUser(user: post.user) { [weak self] _ in
+                    let alertController = UIAlertController(title: "차단 완료", message: "차단되었습니다.", preferredStyle: .alert)
+                    alertController.addAction(UIAlertAction(title: "확인", style: .default) { [weak self] _ in
+                        DispatchQueue.main.async { [weak self] in
+                            guard let self else { return }
+                            communityViewModel.refresh { [weak self] _ in
+                                self?.tableView.refreshControl?.endRefreshing()
+                            }
+                            self.navigationController?.popViewController(animated: true)
```

Comment:
```suggestion
                            communityViewModel.refresh()
```

커뮤니티 화면에서 userBlocked에 있는 `self?.tableView.refreshControl?.endRefreshing()`도 지워주세요.